primeqa/mrc/run_mrc.py

```diff
@@ -205,6 +216,12 @@ class TaskArguments:
     """
     Task specific arguments.
     """
+    modality: str = field(
+        default='table',
```

Comment:
```suggestion
        default='text',
```
The default modality should be text